index.bs

```diff
@@ -2916,6 +2916,252 @@ value and terminate the operation.
 
 </div>
 
+### Signal Credential Changes to the Authenticator - PublicKeyCredential's `signal()` Method ### {#sctn-signal}
+
+<xmp class="idl">
+    partial interface PublicKeyCredential {
+        static Promise<undefined> signal(optional PublicKeyCredentialSignalOptions options = {});
+    };
+
+    dictionary PublicKeyCredentialSignalOptions {
+        UnknownCredentialIdReport              unknownCredential;
+        CurrentCredentialsReport               currentCredentials;
+    };
+
+    dictionary UnknownCredentialIdReport {
+        USVString                              rpId;
+        required Base64URLString               credentialId;
+    };
+
+    dictionary CurrentCredentialsReport {
+        USVString                              rpId;
```

Comment:
Same thing here, better to make it explicit than allow for any potential footguns the optionality might introduce:
```suggestion
        required USVString                              rpId;
```

Reply:
Done.